src/content/configuration/output.md

```diff
@@ -559,6 +559,22 @@ W> Note that if an `array` is provided as an `entry` point, only the last module
 
 T> Read the [authoring libraries guide](/guides/author-libraries/) guide for more information on `output.library` as well as `output.libraryTarget`.
 
+## ouput.scriptType
+
+`string='module'` `bool=false`
```

Comment:
```suggestion
`string: 'module' | 'text/javascript'` `boolean = false`
```

Reply:
Please pay attention to defaults, this option only takes two possible strings, need to list them, also formatting: https://webpack.js.org/contribute/writers-guide/#configuration-defaults-and-types